spec/requests/02_root_route_spec.rb

```diff
@@ -0,0 +1,17 @@
+require "rails_helper"
+
+RSpec.describe "GET /ratingQuestions" do
+let!(:question) do
```

Comment:
Create each of these questions in separate let blocks, or even as a `before` block.

Reply:
do you mean I should have two let here? 
```let!(:question) do
  RatingQuestion.create!(title: "Hello World")
  RatingQuestion.create!(title: "Hello Mars")
end
```
like that?
```let!(:question) do
  RatingQuestion.create!(title: "Hello World")
end
let!(:question) do
  RatingQuestion.create!(title: "Hello Mars")
end
```

Reply:
Yes, but `question_1` and `question_2` as the names, not question.